airflow-core/src/airflow/ui/public/i18n/locales/zh-TW/components.json

```
@@ -49,6 +49,13 @@
     "warning_one": "1 個警告",
     "warning_other": "{{count}} 個警告"
   },
+  "dateRangeFilter": {
+    "validation": {
+      "invalidDateFormat": "無效的日期格式",
+      "invalidTimeFormat": "無效的時間格式",
+      "startBeforeEnd": "開始日期/時間必須在結束日期/時間之前"
```

```suggestion
      "startBeforeEnd": "開始時間必須在結束時間之前"
```

Would it be better this way? "日期/時間" is not that easy to read